pkg/hub/user/identity.go

```diff
@@ -0,0 +1,8 @@
+package user
+
+const (
+	// SubjectPrefix is a prefix for marking open-cluster-management users
+	SubjectPrefix = "system:open-cluster-management:"
+	// SpokeClustersGroup is a common group for all spoke clusters
+	SpokeClustersGroup = SubjectPrefix + "spoke-clusters"
```

Comment:
minor: Use "managedclusters" or "managed-clusters" instead of "spoke-clusters"

Reply:
done, dont have a strong preference between the names tho